dotcom-rendering/src/components/SecureSignup.importable.tsx

```diff
@@ -212,20 +235,20 @@ const sendTracking = (
 };
 
 /**
- * # Secure ReCAPTCHA Signup
+ * # Secure Signup
  *
  * We can only inject ReCAPTCHA client-side, and need to respond to user input.
  *
  * ---
  *
  * [`EmailSignup` on Chromatic](https://www.chromatic.com/component?appId=63e251470cfbe61776b0ef19&csfId=components-emailsignup)
  */
-export const SecureReCAPTCHASignup = ({
-	newsletterId,
-	successDescription,
-}: Props) => {
+export const SecureSignup = ({ newsletterId, successDescription }: Props) => {
 	const recaptchaRef = useRef<ReactGoogleRecaptcha>(null);
 	const [captchaSiteKey, setCaptchaSiteKey] = useState<string>();
+	const [signedInUserEmail, setSignedInUserEmail] = useState<
+		string | undefined
+	>(undefined);
```

Comment:
FYI – types can be inferred

```suggestion
	const [signedInUserEmail, setSignedInUserEmail] = useState<string>();
```